pkgs/development/python-modules/pytest-benchmark/default.nix

```
@@ -33,10 +33,24 @@ buildPythonPackage rec {
   };
 
   patches = [
+    #  replace distutils.spawn.find_executable with shutil.which
```

```suggestion
    # replace distutils.spawn.find_executable with shutil.which
```
nit, just ignore if nothing else will be changed